src/core/Tile/Tile.tsx

```diff
@@ -167,20 +167,22 @@ export const Tile = (props: TileProps) => {
 
           {leftIcon &&
             React.cloneElement(leftIcon as React.ReactElement, {
-              className: 'iui-small iui-type-indicator',
+              className: 'iui-small iui-tile-thumbnail-type-indicator',
```

Comment:
actually, this is more interesting.. we do include buttons and small buttons mixin for `iui-tile-thumbnail-type-indicator`, so there is no need of `iui-small`.. This was forgotten in html though,..There is no need of any button classes, to be honest.
Same with quick action below

Reply:
Removed

Reply:
wait, it's breaking visuals because the button is coming from user who is likely using our `Button`

Reply:
what?
Whats breaking

Reply:
![image](https://user-images.githubusercontent.com/9084735/152543518-5c37367f-add5-4ed0-b6ba-d79c5d97f669.png)

Reply:
because we put users classes after ours :/

Reply:
order of classnames doesnt matter. Order of css imports does, which we cant control

Reply:
we get such a weird situation then.. All the mixins we are reusing in our components might mean nothing in react.. if user is reusing our existing components and the css gets overriden :/